### DISCUSSION TITLE
Abacus for Firefly III does not have local network permissions, resulting in failure to connect to LAN-IP. Please help repair it thank you

### DISCUSSION THREAD

**Discussion by s4w4j4** [victorbalssa/abacus#68] [Bug]
Abacus for Firefly III does not have local network permissions, resulting in failure to connect to LAN-IP. Please help repair it thank you

**Comment by victorbalssa** [comment]
Thank you for your feedback @s4w4j4,
Can you share the URL type of your FireflyIII instance ? You should be able to reach it on your local network even without the local network permission.
Victor

**Reply by s4w4j4** [reply]
URL: http://10.0.0.2:22301/

"Abacus for Firefly III" has no local network permissions and cannot access LAN-IP.
![E7871F03-CB09-40A8-A8F8-0F185320CBC7](https://user-images.githubusercontent.com/90949008/212574218-175ab6a5-3806-4f2a-8bd8-f0b63b61bee2.jpeg)



APP can access LAN-IP only after obtaining local network permissions, such as the screenshot below.
![2E33C348-4B9E-4CA8-B36D-10629C90EEBA](https://user-images.githubusercontent.com/90949008/212574238-fb1c7bbc-4750-49f4-be95-5bcfc502e530.jpeg)

**Reply by victorbalssa** [reply]
@s4w4j4 I'll setup this for the next release `0.5.0`.

**Reply by victorbalssa** [reply]
@s4w4j4 closing this issue, it should work with the `0.5.0`. don't forget to reinstall the app.
